wrappers/Python/CoolProp/__init__.py

```diff
@@ -10,7 +10,7 @@
         os.remove(constants.__file__)
         print("constants shared library has been removed.  Please restart your python code")
     except:
-        print("Unable to remove" + constants.__file__ + ". Please manually remove it")
```

Comment:
Can you please revert the changes in this file?

Reply:
What's wrong here?